web_interface/blueprints/api_v3.py

```
@@ -7577,6 +7577,107 @@ def _validate_starlark_app_path(app_id: str) -> Tuple[bool, Optional[str]]:
+    cmd = [pixlet_path, 'render', str(star_file)]
+    for key, value in pixlet_config.items():
+        if not re.match(r'^[a-zA-Z_][a-zA-Z0-9_]*$', key):
+            continue
+        value_str = 'true' if value is True else 'false' if value is False else str(value)
+        if re.search(r'[`$|<>&;\x00]|\$\(', value_str):
+            continue
+        cmd.append(f'{key}={value_str}')
+    cmd.extend(['-o', output_path, '-m', str(magnify)])
+
+    try:
+        result = subprocess.run(cmd, capture_output=True, text=True, timeout=30, cwd=str(app_dir))
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)